examples/fedpca_examples/dim_reduction/mnist_model.py

```diff
@@ -0,0 +1,17 @@
+import torch
```

Comment:
Not super important, but we've been sticking models that the examples use in `examples/models`